execution-engine/engine-grpc-server/src/engine_server/mappings/state/uref.rs

```
@@ -10,14 +10,15 @@ use crate::engine_server::{
 impl From<AccessRights> for Key_URef_AccessRights {
     fn from(access_rights: AccessRights) -> Self {
         match access_rights {
+            AccessRights::NONE => Key_URef_AccessRights::NONE,
             AccessRights::READ => Key_URef_AccessRights::READ,
             AccessRights::WRITE => Key_URef_AccessRights::WRITE,
             AccessRights::ADD => Key_URef_AccessRights::ADD,
             AccessRights::READ_ADD => Key_URef_AccessRights::READ_ADD,
             AccessRights::READ_WRITE => Key_URef_AccessRights::READ_WRITE,
             AccessRights::ADD_WRITE => Key_URef_AccessRights::ADD_WRITE,
             AccessRights::READ_ADD_WRITE => Key_URef_AccessRights::READ_ADD_WRITE,
-            _ => Key_URef_AccessRights::UNKNOWN,
+            _ => Key_URef_AccessRights::NONE,
```

I think I see the reason that this line is here, although now it may be worth considering removing `_` branch to let the compiler fail once we'd change `AccessRights`

---

Would we even be able to remove it? `AccessRights` is defined with the `bitflags!` macro, and as far as I can tell, it expands to a struct, not an enum - so this would be equivalent to matching on a `u8`, which can have values beyond just the few that are defined as consts, and so this arm might be required. I'm not sure, though.

---

Oh right, my bad. You're right it's an `u8`, so it doesn't make sense to remove this.